enterprise/server/vmexec/vmexec.go

```diff
@@ -360,11 +361,19 @@ func newCommand(start *vmxpb.ExecRequest) (*command, error) {
 	}, nil
 }
 
-func logExecutableInfo(executable string) {
-	path, err := exec.LookPath(executable)
-	if err != nil {
-		log.Warningf("Failed to look up executable in $PATH: %s", err)
-		return
+func logExecutableInfo(workDir string, executable string) {
+	var path string
+	// Relative paths like "tools/foo" or "./foo" are resolved relative to the
+	// working directory. Unqualified executable names like "bash" are resolved
+	// using $PATH lookups.
+	if strings.Contains(executable, "/") {
```

Comment:
nit, because i don't think we actually have a case where we need to care about it, but this should also exclude executables that start with `/`, right?

Reply:
good call, done